webapp/app/dataExport/components/columnSelect/columnSelect.less

```diff
@@ -0,0 +1,7 @@
+.export-column-select {
```

Comment:
don't we have a similar case in country selection?

Reply:
yes

Reply:
can we generalize the style then?

Reply:
yes